### ISSUE TITLE
Hich Contrast theme not set

### ISSUE THREAD

**Issue by jmroyb** [Microsoft/azuredatastudio#4155]
Issue Type: <b>Bug</b>

Hi,

Due my visual impairment I need High Contrast theme or dark themes (my Windows is set as High Contrast). But I have to switch every time I start the app:

1. Press Ctrl+k Ctrl+t
2. Select High Contrast. Watch screen theme changes to black
3. Close and restart app. Watch screen theme is default

Azure Data Studio version: azuredatastudio 1.4.5 (887f4e8985023602e599cf86bdb44df48bb28569, 2019-02-12T06:20:08.633Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 x 3192)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.93GB (8.49GB free)|
|Process Argv|C:\Users\josepm\AppData\Local\Programs\Azure Data Studio\azuredatastudio.exe|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by kburtram** [comment]
@jmroyb I wasn't able to repro this issue locally.  Could you please confirm that you have the following setting in your User Settings file `"workbench.colorTheme": "High Contrast"`?

**Comment by jmroyb** [comment]
Thanks for answer. I did not have this parameter.
